lib/parseMarkdown.js

```diff
@@ -333,6 +333,84 @@ function processData ({ body, attributes }, event, { item, element, combine }) {
         ])
     }
   })
+
+  item('`resources`', () => {
+    if (!attributes.resources) {
+      return element()
+        .condition('no official resources', () => true)
+        .ok(() => {
+          event.resources = []
+        })
+    }
+    element(attributes.resources)
+      .condition('is an array', x => Array.isArray(x))
+      .flatMap(a => combine(a.map(
+        (v, i) => item(`item ${i}`, () => checkValidResource(v))
+      )))
+      .ok(resources => {
+        event.resources = resources
+      })
+    function checkValidResource (v) {
+      const VALID_LINK_TYPES = [
+        'video',
+        'code',
+        'photo',
+        'writeup',
+        'slides'
+      ]
+      return element(v)
+        .conditions([
+          [ '`type` is valid', () => {
+            if (!VALID_LINK_TYPES.includes(v.type)) {
+              return 'type is not valid; allowed types are: ' + VALID_LINK_TYPES.join(', ')
+            }
+            return true
+          } ],
+          [ '`url` is a string', () => typeof v.url === 'string' ],
+          [ '`url` is an http(s) url', () => /^https?:\/\//.test(v.url) ],
+          [ '`detail` is an optional string', () => !v.detail || (typeof v.detail === 'string') ]
+        ])
+    }
+  })
+
+  item('`community_resources`', () => {
+    if (!attributes.community_resources) {
+      return element()
+        .condition('no community resources', () => true)
+        .ok(() => {
+          event.communityResources = []
+        })
+    }
+    element(attributes.community_resources)
+      .condition('is an optional array', x => !x || Array.isArray(x))
+      .flatMap(a => combine(a.map(
+        (v, i) => item(`item ${i}`, () => checkValidCommunityResource(v))
+      )))
+      .ok(communityResources => {
+        event.communityResources = communityResources
+      })
+    function checkValidCommunityResource (v) {
```

Comment:
`checkValidCommunityResource` and `checkValidResource` is exactly the same...

Since `CommunityResource` and `Resource` represents the same concept (“a resource”), it may make sense to deduplicate this.

e.g.

```js
    item('`resources`', () => { ... })
    item('`community_resources`', () => { ... })

    function checkValidResource (v) {
      ...
    }
```